### ISSUE TITLE
Run/Stop button enablement

### ISSUE THREAD

**Issue by ms609** [palaeoware/trevosim#44]
Hi Russell, I've found an hour to start catching up with your recent changes, and have just built the latest version (946b9d031a83f0c6e7caf551bea7401f7401568c).

I've opened a saved settings file (.xml file ending changed to .txt): [tax320_rate2.51.txt](https://github.com/user-attachments/files/15838204/tax320_rate2.51.txt)

When I click "Run" I see the Error "strip uninformative <1, recalculate?", and select "Yes".

- [x] The "Run" button remains illuminated, and "Pause" and "Stop" are disabled.  If I press "Run" again, then the latter buttons light up. Pause seems to work as expected. If I "Stop", then the stop button greys out and the run lights up – but the run continues.  Clicking "Run" again seems to restart the simulation.

- [x] "Reset" seems to reset the current simulation step, but not the entire run.

**Comment by RussellGarwood** [comment]
With above commit, I have fied this issue, which was due to a silly mistake on my part - thanks for highlighting it. I've also made it so the recalculate uninformative functionality works far better if you cancel while it is in the middle of running, so I think this is also an improvement.

Finally, in another commit just before this, I fixed a stupid error I made in 2017 in one of my first bits of coding for TREvoSIm which slowed the gui updates down by ~90%. If I'm being cartiable to myself I could imagine that it was required by the vresion of Qt I was using at the time, but I suspect it was not. As a result, it turns out the GUI updates were actually a good proportion of the run time! Whilst thinking about this, I also modified the software to the GUI only updates a max of every 5 milliseconds. The result is TREvoSim runs much, much faster, and recalculate uninformative factor should be much quicker now. If you are continuing to play with the software, I'd recommend a fresh build. Thanks so much for helping me catch this - it was the root cause of a large number of annoying quirks that didn't impact on the functionality of the simulation, but did have a large impact on the user experience!
